### DISCUSSION TITLE
Any Philippines Site

### DISCUSSION THREAD

**Discussion by lloyde-com** [iiab/iiab#3614] [General]
Hi all
has anyone rolled out any sites in the Philippines.
any issues with the NTC the National Communications Authority - who seems to require a permit for a wide range of devices.
Lloyd

**Comment by holta** [comment]
Hi Lloyd,

In a way XSCE (School Server Community Edition, a.k.a. Internet in a Box) began in the Philippines[*] thanks to the many efforts of Tony Anderson especially, who passed away recently.

I don't know the latest unfortunately, but FYI more remote island areas of the Philippines (like Palawan) were certainly involved.

[*] And also Haiti.
